src/client.ts

```diff
@@ -1642,51 +1659,64 @@ export class Client<
 	 */
 	private async fetch<T = unknown>(
 		url: string,
-		// TODO: Change to `params` when Authorization header support works in browsers with CORS.
-		// _params?: Partial<BuildQueryURLArgs>,
 		params: FetchParams = {},
 	): Promise<T> {
-		// TODO: Restore when Authorization header support works in browsers with CORS.
-		// const accessToken = (params && params.accessToken) || this.accessToken;
-		// const options = accessToken
-		// 	? { headers: { Authorization: `Token ${accessToken}` } }
-		// 	: {};
+		let job: Promise<FetchJobResult>;
 
-		const res = await this.fetchFn(url, {
-			signal: params.signal,
-		});
+		if (this.fetchJobs[url] && this.fetchJobs[url].has(params.signal)) {
+			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+			job = this.fetchJobs[url].get(params.signal)!;
+		} else {
+			this.fetchJobs[url] = this.fetchJobs[url] || new Map();
```

Comment:
Maybe refactor to:

```typescript
this.fetchJobs[url] ||= new Map();
```

Reply:
Actually I saw the 2nd commit my bad. Isn't Babel supposed to do its job here?

Reply:
It's been a while since I made this commit, but I believe this is due to running without a transpiler (i.e. running on Node).

To keep backwards compatibility without much overhead, I think we should keep `||` rather than use `||=`, at least until Node.js >=15 is adopted.